web-embeds/src/App.js

```diff
@@ -15,13 +15,17 @@ Sentry.init({
 function App() {
   const searchElement = document.querySelector('[data-search-feed]');
   const churchElement = document.querySelector('[data-church]');
+  const customPromptElement = document.querySelector('[data-custom-prompt]');
 
   const searchFeed = searchElement
     ? searchElement.getAttribute('data-search-feed')
     : null;
   const church = churchElement
     ? churchElement.getAttribute('data-church')
     : null;
+  const customPrompt = customPromptElement
+    ? customPromptElement.getAttribute('data-custom-prompt')
+    : null;
```

Comment:
Just a suggestion that you might want to avoid `custom` in the data attribute. Since it already has a default value, I think that the nature of this prop being an "override" is apparent enough and it might be more clear/an easier pattern to replicate if it were simply `placeholder` or `data-placeholder` or `data-prompt`.

Just a thought, though!

Reply:
Cool, good idea! Updating